<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <widget class="QWidget" name="searchPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="searchBarHorizontalLayout">
          <item>
           <widget class="QLabel" name="searchLabel">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="searchByComboBox">
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QListView" name="listView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="openRecordButton">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Open Record</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reviseRecordButton">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Revise Record</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="recordButtonVerticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="insertSourcePage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionInsert_New_Source"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInsert_New_Source">
   <property name="text">
    <string>Insert New Source</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
